woocommerce/woocommerce-android
# [Bug] Fix condition for disabling controls in product cards

Closes: #10216 


### Description

This PR disables editable controls in order's product cards in case the order is not editable (e.g. it's in `completed` state, and the lock icon is visible).

### Testing instructions

1. Create order and set it's state to `completed`
2. Open the order and click `Edit`
3. Verify you it's impossible to edit products

### Images/gif

<img src=https://github.com/woocommerce/woocommerce-android/assets/4527432/4bece057-5f7e-4fd6-a200-a86a9d96d24e width=300/>
